e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -336,7 +336,7 @@ describe('Pricing modal', () => {
         cy.get('.CloudUsageModal').contains('Cloud Starter limits');
     });
 
-    it('should open team downgrade selection modal when attempting downgrading from paid to starter', () => {
+    it('should not allow downgrades from enterprise trial', () => {
```

Comment:
Could we add a test for when downgrades are allowed? It seems this test replaces it and we lost that case